homeassistant/components/script.py

```diff
@@ -45,7 +45,7 @@
 })
 
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({cv.slug: SCRIPT_ENTRY_SCHEMA})
+    DOMAIN: cv.schema_with_slug_keys(SCRIPT_ENTRY_SCHEMA)
```

Comment:
This is not the only place that would need this change

```
git grep "{cv.slug:"
```

Reply:
(for a future PR)

Reply:
Indeed there is a ton of them...